### PR TITLE
Fix two bibtex entries.

### PR DESCRIPTION
**Pull request by bangerth**
For one, add a DOI. For the other, remove the 'type' field: There is no
reason why this shouldn't just be printed as a 'PhD thesis'.

/rebuild

### CODE REVIEW

File: doc/doxygen/references.bib [hunk 1]
```diff
@@ -1512,6 +1514,6 @@ @article{Bercovier1979
   number = {2},
   pages = {211--224},
   author = {M. Bercovier and O. Pironneau},
-  title = {Error estimates for finite element method solution of the Stokes problem in the primitive variables},
+  title = {Error estimates for finite element method solution of the {S}tokes problem in the primitive variables},
   journal = {Numerische Mathematik}
 }
```
Comment (marcfehling) [inline doc/doxygen/references.bib:1519]: Do you want to add a newline here? (Does it matter for bibtex?)

File: doc/doxygen/references.bib [hunk 2]
```diff
@@ -122,7 +122,6 @@ @phdthesis{Bec95
   author =  {R. Becker},
   title =   {An Adaptive Finite Element Method for the Incompressible {N}avier--{S}tokes Equations on Time-dependent Domains},
   school =  {Universit{\"a}t Heidelberg},
-  type =    {Dissertation},
   year =    1995
```
Comment (marcfehling) [inline doc/doxygen/references.bib:125]: While there, would you add a link to the thesis as well?
Either https://katalog.ub.uni-heidelberg.de/titel/9608091 or https://www.researchgate.net/publication/2818080

**Review by marcfehling — APPROVED** [review]
Looks good to me! I just have suggestions for further enhancements.

**Comment by bangerth** [comment]
So fixed!

**Review by marcfehling — APPROVED** [review]

**Pull request merged**